hack/integration.sh

```diff
@@ -2,25 +2,36 @@
 
 set -e
 
+REDSKYCTL_BIN="${REDSKYCTL_BIN:=dist/redskyctl_linux_amd64/redskyctl}"
```

Comment:
Would this be more useful for running locally if it were: `REDSKYCTL="${REDSKYCTL:-redskyctl}"` And then the CI action explicitly set the path to the full value?

Reply:
I went this way since the main use is for CI, but allows us to do `REDSKYCTL_BIN=$(which redskyctl) hack/integration.sh` locally ( what I did to test this :D )